cactusbot/handlers/command.py

```diff
@@ -128,6 +128,11 @@ def __init__(self, channel, api):
         await self.api.update_command_count(command, "+1")
         if not is_alias and "count" not in data:
             data["count"] = str(json["count"] + 1)
+        elif is_alias:
+            command_data = (await (await self.api.get_command(
```

Comment:
Add a check to make sure the response code is `200`. Otherwise we could have something go wrong in here and then there's no response and an uncaught error.

```python
cmd = await self.api.get_command(...)
if cmd.status != 200:
    return "explosions occureth"
```